src/Chronos.php

```diff
@@ -1009,12 +1009,16 @@ public function setTimezone(DateTimeZone|string $value): static
     /**
      * Return time zone set for this instance.
      *
-     * @return \DateTimeZone|null
+     * @return \DateTimeZone
      */
-    #[ReturnTypeWillChange]
-    public function getTimezone(): ?DateTimeZone
+    public function getTimezone(): DateTimeZone
     {
-        return parent::getTimezone() ?: null;
+        $tz = parent::getTimezone();
+        if ($tz === false) {
+            throw new RuntimeException('Time zone could not be retrieved.');
```

Comment:
I don't think it's possible to trigger this and just reflects the internal timelib functions. Maybe if you serialized DateTimeImmutable with a time zone that was later removed? I'm not sure.

Reply:
Meh, I wouldn't worry about getting coverage on this case.